app/assets/stylesheets/frontend/views/_history-people.scss

```diff
@@ -36,28 +40,13 @@
   }
 
   .foreign-secretaries {
```

Comment:
It looks like you're having to fight quite a lot with some existing styles to make the govuk-grid work here, which I sympathise with, but it still doesn't quite work at some smaller screen sizes.

<img width="636" alt="Screenshot 2019-10-25 at 10 23 33" src="https://user-images.githubusercontent.com/861310/67559902-ba15b400-f711-11e9-913b-9df54a720a30.png">

Probably not worth it for this PR, but might be good to consider dropping this layout altogether and simply putting the heading above the list of names and moving the list of names into a 4 column layout like the section above.

Reply:
Changed to a four column layout 👍